File: api/src/main/kotlin/api/db/models/Calendar.kt
package api.db.models

data class Calendar(
    val serviceId: String,
    val monday: Boolean,
    val tuesday: Boolean,
    val wednesday: Boolean,
    val thursday: Boolean,
    val friday: Boolean,
    val saturday: Boolean,
    val sunday: Boolean,
    val startDate: Long,
    val endDate: Long,
)

File: MadridTransporte.Api/Data/Entities/Calendar.cs
namespace MadridTransporte.Api.Data.Entities;

public class Calendar
{
    public int Id { get; set; }
    public required string ServiceId { get; set; }
    public bool Monday { get; set; }
    public bool Tuesday { get; set; }
    public bool Wednesday { get; set; }
    public bool Thursday { get; set; }
    public bool Friday { get; set; }
    public bool Saturday { get; set; }
    public bool Sunday { get; set; }
    public long StartDate { get; set; }
    public long EndDate { get; set; }
}
